<launch>
	<master auto="start"/>
	<param name ="/use_sim_time" value="true"/>

	<node name="convert" pkg="projeto_semear" type="convert_colorsensor" respawn="false" output="screen"/>

	<node name="navigating" pkg="projeto_semear" type="navigating_server" respawn="false" output="screen"/>

	<node name="path_planning" pkg="projeto_semear" type="path_planning_server" respawn="false" output="screen"/>

	<node name="gps" pkg="projeto_semear" type="gps" respawn="false" output="screen"/>
	
	<node name="mapa" pkg="projeto_semear" type="mapa_container_server" respawn="false" output="screen"/>

	<node name="eletroima" pkg="projeto_semear" type="eletroima_control_server" respawn="false" output="screen"/>

	<node name="descobrir_cor_srv" pkg="projeto_semear" type="descobrir_cor" respawn="false" output="screen"/>

</launch>   
